examples/creatives/nonlinear_overlay.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * A sample SIMID nonlinear ad that shows an overlay of
+ * buttons to showcase functionality
+ */
+class NonLinear extends BaseSimidCreative {
```

Comment:
Let's call this something other than nonlinear, then name the class similarily to how you name the file.